cumulusci/tasks/robotframework/robotframework.py

```diff
@@ -97,13 +99,20 @@ def _run_task(self):
         )
 
         if self.options["processes"] > 1:
+            # Since pabot runs multiple processes, we have to use the
+            # --pythonpath option to make sure the current working
+            # directory is included when pabot spawns each
+            # process. Otherwise, robot can't file libraries and resource
+            # files via relative paths.
```

Comment:
@boakley Can you show me the symptom you're trying to address?

At first glance, it doesn't make sense to me that this would be the solution. Python typically does include the current directory in sys.path without needing to be explicit about it.